back/engines/commercial/insights/spec/acceptance/views_spec.rb

```diff
@@ -117,6 +122,19 @@
         expect(json_response).to match(expected_response)
       end
 
+      example_request 'copies topic to assignments' do
+        view =  Insights::View.find(json_response[:data][:id])
+        aggregate_failures 'check assignments' do
+          expect(status).to eq(201)
+          expect(
+            assignment_service
+              .approved_assignments(ideas[1], view).pluck(:category_id)
+              .length
+          ).to eq(2)
+          expect(view.categories.length).to eq(3)
```

Comment:
Just a suggestion, but I like to put the status-code check outside of the aggregation, bc the other expectation will not be interesting if the request fails or has an unexpected status code.

```suggestion
        expect(status).to eq(201)
        
        aggregate_failures 'check assignments' do
          expect(view.categories.length).to eq(3)
          expect(assignment_service.approved_assignments(ideas[1], view).count).to eq(2)
```